apps/munimoney/README.md

```diff
@@ -0,0 +1,24 @@
+### Steps to deploy Munimoney
+_____________________________
+
+*for more detail see README of [OpenUpSA/ansible-config](https://github.com/OpenUpSA/ansible-config) repo's root
+  - Run the following Ansible playbooks from root directory a local checkout of the `OpenUpSA/ansible-config` repo
+  - ensure you `pass git pull`
+  - make use of appropriate inventory flag depending on environment and substitute out `staging` for correct `env_name` as necessary
+
+1. Configure users of the new host*
+   - `ansible-playbook -i inventory/staging.yml --limit munimoney users.yml --user root`
+2. Configure Dokku on the new host*
+   - `ansible-playbook -i inventory/staging.yml --limit munimoney dokku-server.yml`
+3. Configure mail server for forwarding cron errors on the new host*
+   - `ansible-playbook -i inventory/staging.yml --limit munimoney ssmtp.yml -e "ssmtp_AuthUser=apikey ssmtp_AuthPass=$(pass show services/sendgrid.net | head -n 1) ssmtp_root=webapps@openup.org.za crond_MAILTO=webapps@openup.org.za"`
```

Comment:
I don't think each app's docs needs to say how to set up a new server - the root readme says that. You can perhaps say something like "This assumes thatthe server has been configured as per the [standard server configuration](../../README.md)"